src/generators.ts

```diff
@@ -83,7 +83,16 @@ export class FolderGenerator<T> implements IGenerator<T> {
     innterContext.workingDirectory = path.join(context.workingDirectory, folderName);
     innterContext = resolveContextModifier(innterContext, this._contextMap);
 
-    await fs.mkdir(innterContext.workingDirectory);
+    if((context.options as any).overWriteDir === true) {
+      await fs.rmdir(innterContext.workingDirectory);
+      await fs.mkdir(innterContext.workingDirectory);
+    } else if((context.options as any).overWriteDir === false) {
+      console.log('Aborting');
+      process.exit(0)

```

Comment:
may be it would be better return `1` as error?

Reply:
This is the output with `1` as error code:
![image](https://user-images.githubusercontent.com/22073083/51174692-71e77c80-18c9-11e9-80c1-cf089d21f376.png)
With `0` as an error code there is no output on process exit:
![image](https://user-images.githubusercontent.com/22073083/51174780-b07d3700-18c9-11e9-9b36-19bd3b04fb13.png)


In my opinion the user doesn't need to see any otuput, because there is actually no error - the process is simply aborted because we don't overwrite directory -> can't assemble the plugin.

@amper43 please advise - leave `0` or use `1` instead?

Reply:
@sankerust you're right, `0` looks much better